node/src/Worker.ts

```diff
@@ -175,15 +175,15 @@ export type WorkerResourceUsage =
 	/* eslint-enable camelcase */
 };
 
-export type WorkerEvents = 
-{ 
+export type WorkerEvents =
+{
 	died: [Error];
 	// Private events.
 	'@success': [];
 	'@failure': [Error];
 };
 
-export type WorkerObserverEvents = 
+export type WorkerObserverEvents =
```

Comment:
I fail to understand why my editor is not removing those spaces automatically on save. It usually does. Good catch.

Reply:
Give thanks to VS Code, not me :-)

Reply:
I expected our ESLint tules to catch them but they don't.

Reply:
Yeah, that surprised me a bit too.